spec.md

```diff
@@ -640,9 +640,9 @@ did not include a `"requires":["route_forwarding"]` property.
 
 There are a class of services that provide network storage to applications
 via volume mounts in the application container. A create binding response
-from one of these services MUST include a `volume_mount`.
+from one of these services MUST include `volume_mounts`.
```

Comment:
I'm cool with moving this information in, but we might want to add a note saying that for route and volume services, users should check to see that their platforms support those services.